test/factories/host_related.rb

```diff
@@ -299,6 +299,17 @@ def set_nic_attributes(host, attributes, evaluator)
       subnet
     end
 
+    trait :with_rootpass do
+      root_pass 'xybxa6JUkz63w'
+    end
+
+    trait :with_os do
```

Comment:
That's misleading too as I don't (know how to) enable subnet/domain. What if I split this one by one? `:with_arch`, `:with_medium` etc ... ?